packages/frontend/src/graphql/queries/get-apps.ts

```diff
@@ -20,6 +20,8 @@ export const GET_APPS = gql`
       connectionCount
       description
       isNewApp
+      connectionStepLabel
```

Comment:
not sure if i'm missing something, but you're already replacing the substep name in the backend. why is this needed?

Reply:
Discussed - this is to support different labels depending on the event chosen